3/debian-10/rootfs/opt/bitnami/scripts/libcassandra.sh

```diff
@@ -33,6 +33,7 @@ export CASSANDRA_BIN_DIR="${CASSANDRA_BASE_DIR}/bin"
 export CASSANDRA_CONF_DIR="${CASSANDRA_BASE_DIR}/conf"
 export CASSANDRA_VOLUME_DIR="${CASSANDRA_VOLUME_DIR:-/bitnami/cassandra}"
 export CASSANDRA_DATA_DIR="${CASSANDRA_DATA_DIR:-${CASSANDRA_VOLUME_DIR}/data}"
+export CASSANDRA_COMMITLOG_DIR="{CASSANDRA_COMMITLOG_DIR:-${CASSANDRA_VOLUME_DIR}/data/commitlog}"
```

Comment:
There is a `$` missing here:

```suggestion
export CASSANDRA_COMMITLOG_DIR="${CASSANDRA_COMMITLOG_DIR:-${CASSANDRA_VOLUME_DIR}/data/commitlog}"
```